### DISCUSSION TITLE
oen shell on windows start up

### DISCUSSION THREAD

**Discussion by Murf1954** [Open-Shell/Open-Shell-Menu#1517] [General]
I amusing open shell 4.4.189 Love it  my question is every time  i hit restart or start at boot up the shell start up does not come up right away  i get the  windows 10 start up start button  but eventually it  does   How do  the start button the shel button to open automatically at  start up. Instead of waiting seconds for it to come up?

**Comment by tallman69** [comment]
How long are you having to wait from the time your desktop screen displays to the time the shell button loads?  My desktop computer boots up with the shell button in place.  But for my two laptops, I have to wait 2-3 seconds for the shell button to load on the one, and about 5-6 seconds for the shell button to load on the other.

**Comment by ge0rdi** [comment]
@Murf1954 
You may try to reinstall `Open-Shell`.
It is supposed to start right when Explorer starts (so you should not see original Windows start menu at any point).

In your case it seems that it won't start with Explorer (for some reason).
Then we have also item registered in standard _startup_ location, but those locations are processed only some time after login (that's why it starts working for you after some time). Though this mechanism is meant only as a backup.

**Reply by tallman69** [reply]
I have always thought that the delay was normal for slower machines..  I have disabled StartMenu.exe in the Startup tab of Task Manager to see if Open-Shell was loading from there, but Open-Shell loaded with it's normal delay on starup.  I then uninstalled and reinstalled Open-Shell which did not change the behavior any.

**Reply by ge0rdi** [reply]
> I have always thought that the delay was normal for slower machines.

Hmm, what delay you mean exactly?

Is it a delay between you press Start button for the first time and `Open-Shell` menu shows?
Or you can trigger original Windows start menu shortly after logon?

**Reply by tallman69** [reply]
Murf1954 commented that the Open-Shell start button was slow to load in place of the default Windows start button.  So I was simply measuring how long it took for the Open-Shell start button to load after the Windows start button appeared.  I am not able to launch the default Windows start menu before the Open-Shell button loads.  If I click on the Windows start button I get a message:  **Open-Shell Menu**, Please wait for the start menu to finish the initial collection of information. The Open-Shell shell start menu then launches several seconds after the message appears.
